src/components/android-settings.js

```diff
@@ -5,6 +5,7 @@ import MenuItem from '@material-ui/core/MenuItem'
 import Radio from '@material-ui/core/Radio'
 import RadioGroup from '@material-ui/core/RadioGroup'
 import FormControlLabel from '@material-ui/core/FormControlLabel'
+import style from '../styles/settings.style'
```

Comment:
This should probably use CSS Modules instead of inline styles!

https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Reply:
@Sharmyn28 reminding you about this comment, still unaddressed in this PR